3회차/오채현/4_신용카드만들기1.py

```diff
@@ -1,3 +1,41 @@
 import sys
 
 sys.stdin = open("_신용카드만들기1.txt")
+
+
+# testCase = [4,5,2,0,0,2,0,0,1,9,0,0,4,1,6]
+# testCase = [4, 5, 2, 0, 0, 2, 0, 0, 1, 9, 0, 0, 4, 0, 6]
+
+
+# total = 0
+
+# for idx in range(1, len(testCase)+1):
+#     if idx % 2 == 0:
+#         total += testCase[idx-1]
+#     else:
+#         total += testCase[idx-1] * 2
+
+# n = 10 - (total % 10)
+# print(f'# {n}')
+
+T = int(input())
+# 여러개의 테스트 케이스가 주어지므로, 각각을 처리합니다.
+for test_case in range(1, T + 1):
+    
+    nums = list(map(int, input().split()))
+    total = 0
+
+    for idx in range(1, len(nums)+1):
+        if idx % 2 == 0:
+            total += nums[idx-1]
+        else:
+            total += nums[idx-1] * 2
+
+#인덱스 문제는 아님..
+
+    if total == 0:
+        n = 10
+    else:
+        n = 10 - int(total % 10)
```

Comment:
규칙에 맞게 `total` 합을 잘 구해주셨네요 😄 
정답 출력을 위해 고민하신 것 같은데 조건을 아쉽게도 잘못 나눠주셨어요.
`total`을 10으로 나눈 나머지가 0일 때, 마지막 카드 자리는 0이 되어야겠죠?
```suggestion
    if total % 10 == 0:
        n = 0
    else:
        n = 10 - int(total % 10)
```